src/styles/global.css

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 
 :root {
   --cpd-radius-pill-effect: 9999px;
+
+  /* Default icon and avatar size */
+  --cpd-icon-button-size: var(--cpd-space-8x);
+  --cpd-avatar-size: var(--cpd-space-16x);
```

Comment:
This is mainly to avoid the "variable is not defined" error for those, even though they are always set within the component